donkeycar/parts/datastore.py

```diff
@@ -235,6 +235,9 @@ def put_record(self, data):
                 name = self.make_file_name(key, ext='.png')
                 img.save(os.path.join(self.path, name))
                 json_data[key]=name
+                
+            elif typ == 'nparray':                         # if array type
+                json_data[key] = (np.array(val)).tolist()  # store the input as an array into tub
```

Comment:
Same comment as below - if `type(val) == np.array` then we shouldn't need to wrap it into `(np.array(val))`, so `val.tolist()` should work.